tests/test_execute.py

```diff
@@ -181,14 +181,27 @@ def test_execute_notebook_remove_tagged_cells(tmp_empty):
     assert [c.source for c in out.cells] == ["1 + 1"]
 
 
-def test_execute_notebook_save_profiling_data(tmp_empty):
+@pytest.mark.parametrize(
+    "save_profiling_data_value, save_profiling_output",
+    [(True, "out-profiling-data.csv"), ("test.csv", "test.csv")],
```

Comment:
here it makes sense to test with other data types since they'll be passed to the function (floats, dict)

Reply:
Got you

Reply:
Add other test cases in [a851050](https://github.com/ploomber/ploomber-engine/pull/78/commits/a851050318c7185a5e28ae4508f12b738ce815c3)